godotproject/Menus/ThanksForPlayingScreen/ThanksForPlayingScreen.gd

```diff
@@ -5,13 +5,13 @@ var inhibit_hud = true
 var start = true
 var scroll_speed = 0.5
 
-
 func _process(_delta):
 	if(start == true):
 		start = false
 		$VBoxContainer2.rect_position.y = 500
+		self.margin_top = 0
```

Comment:
In the scene file, I set the MarginContainer's top margin to 512 so that all the text is initially off-screen. It seems to show up for a split second unless we do this.